dmwmclient/rucio.py

```diff
@@ -179,29 +216,85 @@ async def list_content(self, scope, name):
         scope = quote(scope, safe="")
         name = quote(name, safe="")
         method = "/".join(["dids", scope, name, "dids"])
-        return await self.getjson(method)
+        data = await self.getjson(method)
+        out = []
+        for key in data:
+            out.append(
+                {
```

Comment:
The first times I run the script using the dmwm shell, I used the original dictionary. But, for some reason it was really complicated to access the right key and the right value. Plus, json_normalize(old_dictionary) produced a very confusing df with little sense. I had to create a new plain dictionary so it could work.

Reply:
Ok, I would have thought the old and new dictionary here are almost identical, but maybe I am missing something